src/WebJobs.Script/Utility.cs

```diff
@@ -924,28 +924,32 @@ public static bool IsMediaTypeOctetOrMultipart(MediaTypeHeaderValue mediaType)
             }
         }
 
-        public static bool CanWorkerIndex(IEnumerable<RpcWorkerConfig> workerConfigs, IEnvironment environment)
+        public static bool CanWorkerIndex(IEnumerable<RpcWorkerConfig> workerConfigs, IEnvironment environment, FunctionsHostingConfigOptions functionsHostingConfigOptions)
```

Comment:
The logic of `CanWorkerIndex` is getting complicated with lot of checks. Can we add a comment explaining different conditions (similar to description of this PR)?

Reply:
done